.github/workflows/pypi.yml

```diff
@@ -23,32 +23,38 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
-      - if: ${{ matrix.os == 'ubuntu-20.04' }}
+
+      # TODO: support non-x86_64 platforms
+      - if: ${{ (matrix.os == 'ubuntu-20.04') && (matrix.python-version == '3.6') }}
         run: |
-          wget -O - https://downloads.sourceforge.net/swig/swig-4.0.1.tar.gz | tar xzf -
-          cd swig-4.0.1 && ./configure --without-pcre && make && sudo make install
+          wget -O - https://downloads.sourceforge.net/swig/swig-4.0.1.tar.gz | tar xzf - &&
+          docker run --rm -e PLAT=manylinux1_x86_64 -v `pwd`:/io quay.io/pypa/manylinux1_x86_64 /io/travis/build-wheels.sh
+
       - if: ${{ matrix.os == 'windows-latest' }}
         run: |
           choco install swig
       - if: ${{ matrix.os == 'macos-latest' }}
         run: |
           brew update
           brew install swig
-      - run: python -m pip install build
-      - run: pip install -r requirements.txt
-      - run: pip install -r dev-requirements.txt
-      - run: pip install .
-      - run: make travis-test
-      - name: Build a binary wheel and a source tarball
+
+      # windows/mac builds
+      - if: ${{ matrix.os != 'ubuntu-20.04' }}
+        run: python -m pip install build
+      - if: ${{ matrix.os != 'ubuntu-20.04' }}
+        run: pip install -r requirements.txt
+      - if: ${{ matrix.os != 'ubuntu-20.04' }}
+        run: pip install -r dev-requirements.txt
+      - if: ${{ matrix.os != 'ubuntu-20.04' }}
+        run: pip install .
+      - if: ${{ matrix.os != 'ubuntu-20.04' }}
+        run: make travis-test
+      - if: ${{ matrix.os != 'ubuntu-20.04' }}
+        name: Build a binary wheel and a source tarball
```

Comment:
that's a lot of 'if's :)
I wonder if it's possible to merge then either into single statement or perhaps even better into another (build_wheels_win.sh) script? Up to you, if you are up to it. Other than that feel free to merge, you should be able to do it.

Reply:
good point ;)